lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1561,10 +1582,10 @@ def _on_move(self, event):
                 return
 
             # Convert to quaternion
-            elev = np.deg2rad(self.elev)
             azim = np.deg2rad(self.azim)
+            elev = np.deg2rad(self.elev)
             roll = np.deg2rad(self.roll)
-            q = _Quaternion.from_cardan_angles(elev, azim, roll)
+            q = _Quaternion.from_cardan_angles(azim, elev, roll)
```

Comment:
...well actually, the order _is_ the order that the _Quaternion API defines, and the change is as intended. Since the quaternion class has its own API, it is not bound by my previous mistake to force it to be consistent with the Axes3D class. And furthermore, _Quaternion is an internal class, so I have little hesitation to change it back to the way it was initially, with a logical progression of arguments for `from_cardan_angles()` and `as_cardan_angles()`.